Examples/security_rules.ruleset

```diff
@@ -0,0 +1,135 @@
+let backup_retention = 7
+let allowed_ports = [80, 443]
+
+AWS::AmazonMQ::Broker AutoMinorVersionUpgrade == true << Version upgrades should be enabled to receive security updates
+AWS::AmazonMQ::Broker EncryptionOptions.UseAwsOwnedKey == false << CMKs should be used instead of AWS-provided KMS keys
+AWS::AmazonMQ::Broker EngineVersion == 5.15.10 << Broker engine version should be at least 5.15.10
+AWS::AmazonMQ::Broker Logs.Audit == true << Audit logging should be enabled
+AWS::AmazonMQ::Broker Logs.General == true << General logging should be enabled
+AWS::AmazonMQ::Broker PubliclyAccessible == false << Broker should not be publicly accessible
+
+AWS::AmazonMQ::Configuration EngineVersion == 5.15.10 << Broker engine version should be at least 5.15.10
```

Comment:
[`MQ 5.15.12`](https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/broker-engine.html) seems even better, but is not allowed by these rules. The last character should probably be a wildcard